airbyte-integrations/connectors/source-google-ads/source_google_ads/schemas/keyword_report.json

```diff
@@ -67,6 +67,9 @@
     "segments.date": {
       "type": ["null", "string"],
       "format": "date"
+    },
+     "segments.device": {
+      "type": ["null", "string"]
```

Comment:
If I'm not wrong the field `segments.device` doesn't work with other `segments` fields.

Reply:
I thikk there is no problem. Attached doc
![image](https://user-images.githubusercontent.com/74010842/205354230-1be0d9a1-bfd8-4497-a42b-88f8e3ae3a3b.png)
https://developers.google.com/google-ads/api/fields/v12/keyword_view?hl=en

Reply:
Found the problem and updated google_ads keyword_report.json file